bitnami/nginx/README.md

```diff
@@ -125,6 +125,11 @@ The following tables lists the configurable parameters of the NGINX Open Source
 | `metrics.serviceMonitor.interval`          | Interval at which metrics should be scraped.                                                | `nil` (Prometheus Operator default value)                    |
 | `metrics.serviceMonitor.scrapeTimeout`     | Timeout after which the scrape is ended                                                     | `nil` (Prometheus Operator default value)                    |
 | `metrics.serviceMonitor.selector`          | Prometheus instance selector labels                                                         | `nil`                                                        |
+| `autoscaling.enabled`                      | Enable autoscaling for NGINX deployment                                                     | `false`                                                      |
+| `autoscaling.minReplicas`                  | Minimum number of replicas to scale back                                                    | `nil`                                                        |
+| `autoscaling.maxReplicas`                  | Maximum number of replicas to sacle out                                                     | `nil`                                                        |
```

Comment:
There is a typo here: `sacle` -> `scale`

Reply:
Typo fixed